force_wfmanager/ui/setup/mco/mco_view.py

```diff
@@ -0,0 +1,92 @@
+from traits.api import (
+    Instance, List, Unicode, on_trait_change, Bool, Event,
+    HasTraits, Either
+)
+from traitsui.api import View
+
+from force_bdss.api import BaseMCOModel
+from force_wfmanager.ui.setup.mco.kpi_specification_view import \
+    KPISpecificationView
+from force_wfmanager.utils.variable_names_registry import \
+    VariableNamesRegistry
+
+from .mco_parameter_view import MCOParameterView
+from force_wfmanager.ui.ui_utils import get_factory_name
+
+
+class MCOView(HasTraits):
+
+    # -------------------
+    # Required Attributes
+    # -------------------
+
+    #: MCO model (More restrictive than the ModelView model attribute)
+    model = Instance(BaseMCOModel)
+
+    #: Registry of the available variables
+    variable_names_registry = Instance(VariableNamesRegistry)
+
+    # ------------------
+    # Regular Attributes
+    # ------------------
+
+    #: List of MCO parameter and KPI views to be displayed in the TreeEditor
+    #: NOTE: (Has to be a list to be selectable in TreeEditor)
+    mco_options = List(Either(Instance(MCOParameterView),
```

Comment:
This list contains single instances of both `MCOParameterView` and `KPISpecificationView`, which are displayed in the `TreeEditor`. They are in the same trait `mco_options`, since this is referenced in the MCO's `TreeNode` as the `children` list trait it displays.

Reply:
We should document the form of this attribute, from the two listeners below it looks like it's a 2 element list with `[MCOParameterView, KPISpecificationView]`